content/introduction/introduction_notation.rst

```diff
@@ -26,7 +25,7 @@ Integrals
     .. math::
         \int_V f ~dv
 
-   or over a closed volume
+   or over a closed volume:
```

Comment:
Correct me if I'm wrong, but I don't think such an integral exist, i.e., over a closed volume (what does that mean?)